Practical Example/IosInterviewPracticeProject/IosInterviewPracticeProject/Constants/ContentUrls.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  ContentUrls.swift
+//  IosInterviewPracticeProject
+//
+//  Created by Robert Ramirez on 11/21/20.
+//  Copyright © 2020 me.robert.ramirez. All rights reserved.
+//
+
+import Foundation
+
+let articleUrl = "https://raw.githubusercontent.com/raywenderlich/ios-interview/master/Practical%20Example/articles.json"
+let videoUrl = "https://raw.githubusercontent.com/raywenderlich/ios-interview/master/Practical%20Example/videos.json"
```

Comment:
Just a minor note on naming convention. In Swift acronyms that are usually uppercase like URL are kept uppercase in names. So these names should be `articleURL` and `videoURL` if you want to be consistent with Apple's APIs.